pages/profile.tsx

```diff
@@ -58,17 +63,18 @@ export const ProfilePage: NextPage = () => {
                 variant="line"
                 colorScheme="black"
                 size="lg"
+                onChange={(index) => setTabIndex(index)}
               >
                 <TabList color="black">
-                  <Tab>My Collections</Tab>
-                  <Tab>Profile</Tab>
+                  <Tab _selected={{ fontWeight: 700 }}>My Collections</Tab>
+                  <Tab _selected={{ fontWeight: 700 }}>Profile</Tab>
                 </TabList>
-
-                <TabPanels>
-                  <TabPanel></TabPanel>
-                  <TabPanel></TabPanel>
-                </TabPanels>
               </Tabs>
+
+              <VStack pt={{ base: 2, md: 4 }} alignItems="left" w="100%">
+                {tabIndex === 0 && <Collections userData={userData} />}
+                {tabIndex === 1 && <Text> Profile </Text>}
+              </VStack>
```

Comment:
Can't you just put the contents of this `VStack` in the `TabPanels`? That's kinda the whole point of the `Tabs` component?

Reply:
That's a good point I guess I didn't consider that because I just wanted to have something handle the actual tabbing